service/dap/server.go

```diff
@@ -16,12 +16,13 @@ import (
 	"net"
 	"path/filepath"
 
+	"github.com/go-delve/delve/pkg/gobuild"
 	"github.com/go-delve/delve/pkg/logflags"
 	"github.com/go-delve/delve/pkg/proc"
 	"github.com/go-delve/delve/service"
 	"github.com/go-delve/delve/service/api"
 	"github.com/go-delve/delve/service/debugger"
-	"github.com/google/go-dap"
+	"github.com/google/go-dap" // dap
```

Comment:
I'm not sure what's adding this, I just recently removed it - this comment is not needed

Reply:
I believe Hana added this because the import ends with go-dap, but when you reference the items in the package, you prefix them with "dap".

Reply:
I don't think this comment is semantically necessary, though. [one of my previous commits just removed it -- https://github.com/polinasok/delve/commit/23b8e59ab9837698221eb50fadcf6c5fcce2d1a7 -- so it'd be a shame to add it back]

It's perfectly fine in go to have an importa path "like-this" and referenced as "this", and many examples in Delve itself. E.g. see usage of `go-colorable` in https://github.com/go-delve/delve/blob/master/pkg/terminal/terminal_windows.go#L9

Reply:
Removed. I have no attachment to this comment. I thought it was some kind of convention. It was still in my file as I copied it around trying to avoid conflicts with your commits.